ontotext-graphql-playground-component/src/components/graphql-playground-component/graphql-playground-component.tsx

```diff
@@ -73,15 +76,22 @@ export class GraphqlPlaygroundComponent {
 
   private init(configuration: ExternalGraphqlPlaygroundConfiguration) {
     const containerEl = document.querySelector('graphql-playground-component #graphiql');
-    // @ts-ignore
-    const root = ReactDOM.createRoot(containerEl);
+    if (!containerEl) {
+      console.error('Container element not found');
+      return;
+    }
+    if (!this.reactRoot) {
+      // Create the root only once
+      // @ts-ignore
+      this.reactRoot = ReactDOM.createRoot(containerEl);
```

Comment:
I just noticed -  we have to unmount this on disconnectedCallback. Otherwise I think it will leak.